docs/structure.ts

```diff
@@ -383,6 +383,17 @@ export const STRUCTURE = [
           },
         ],
       },
+      {
```

Comment:
Also, let's add the component to the https://akveo.github.io/nebular/#/docs/concepts/ui-kit page